src/components/Comments/ReplyCommentItem.js

```diff
@@ -316,11 +379,18 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+        {<Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
         <TouchableOpacity onPress={onUpVote} testID="upvotebtn" activeOpacity={1}>
           <ButtonHightlight
             onLongPress={handleLongPress}
-            style={[styles.arrowdown, styles.btn]}
+            style={[
+              styles.arrowdown,
+              styles.btn,
+              {
+                paddingLeft: 0,
+                marginLeft: 8
+              }
+            ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styling for the upvote button has been updated. Make sure that these changes look good across different screen sizes and orientations.


<!-- This is an auto-generated comment by OSS CodeRabbit -->